<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Result_Window</class>
 <widget class="QDialog" name="Result_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #6c7a89, stop: 1 #bdc3c7);</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="Fromlabel">
     <property name="styleSheet">
      <string notr="true">
font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>From:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="FromResult">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="Tolabel">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>To:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="ToResult">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Pricelabel">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Price:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="PriceResult">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="Option1label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Option1:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="Option1Result">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="Option2label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Option2:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="Option2Result">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="Option3label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Option3:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="Option3Result">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="Option4label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Option4:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="Option4Result">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="Option5label">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>Option5:</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="Option5Result">
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Segoe UI&quot;;
background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0">
    <widget class="QToolButton" name="toolButton">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QToolButton" name="backtoolButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
